app/apollo/models/resource.schema.js

```diff
@@ -43,13 +43,20 @@ const ResourceSchema = new mongoose.Schema({
     default: false,
   },
   searchableData: {
-    type: Map,
+    type: Object,
```

Comment:
What's the implication of changing Map to Object?  This seems like it should be extraneous to adding `/v3/:cluster_id/resources` and adding to GQL.